MediaBrowser.MediaEncoding/Subtitles/SubtitleEditParser.cs

```diff
@@ -57,5 +85,94 @@ public SubtitleTrackInfo Parse(Stream stream, CancellationToken cancellationToke
             trackInfo.TrackEvents = trackEvents;
             return trackInfo;
         }
+
+        /// <inheritdoc />
+        public bool SupportsFileExtension(string fileExtension)
+            => _subtitleFormats.ContainsKey(fileExtension);
+
+        private IEnumerable<SubtitleFormat> GetSubtitleFormats()
+        {
+            var subtitleFormats = new List<SubtitleFormat>();
+            var assembly = Assembly.GetAssembly(typeof(SubtitleFormat));
```

Comment:
```suggestion
            var assembly = typeof(SubtitleFormat).Assembly;
```
https://docs.microsoft.com/en-us/dotnet/api/system.reflection.assembly.getassembly?view=net-7.0#remarks